src/controllers/articleController.js

```diff
@@ -0,0 +1,91 @@
+import { User, Article } from '../sequelize/models';
+import { slugGen, uploadImage } from '../helpers/articles/articleHelper';
+
+/**
+ * @description holds article logic
+ */
+class ArticleController {
+  /**
+   * @description create article
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} returns object with created article
+   */
+  static async createArticle(req, res) {
+    try {
+      const { id } = req.userData;
+      const author = await User.findOne({ where: { id } });
```

Comment:
a person is able to create an article even though he's not verified. So I think to be in the database is not enough, you should also check if he's verified, then allow him to create an article

Reply:
I think that is impossible! So, How did you get the token? Because the route for creating an article is protected.

Reply:
The token is only provided after login and the user will have already verified the account.

Reply:
You're Right!

Reply:
yeah true, they corrected that. because when I signup I was getting the token too